src/Nox.Solution/Nox.Solution.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;net7.0</TargetFrameworks>
```

Comment:
Need to find the way how to use netstandard2.0 target only in Nox.Generator and keep net7.0 here